helm/values.yaml

```diff
@@ -3,7 +3,20 @@ global:
   tracing: {}
   metrics: {}
   environment: ''
-
+  storage:
```

Comment:
is this section duplicated with line 11?

Reply:
as global mechanism, one is local level and one is global